Sources/StreamChat/Models/Attachments/ChatMessageAudioAttachment.swift

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A type alias for attachment with `AudioAttachmentPayload` payload type.
+///
+/// The `ChatMessageAudioAttachment` attachment will be added to the message automatically
+/// if the message was sent with attached `AnyAttachmentPayload` created with
+/// local URL and `.audio` attachment type.
+public typealias ChatMessageAudioAttachment = _ChatMessageAttachment<AudioAttachmentPayload>
+
+/// Represents a payload for attachments with `.media` type.
+public struct AudioAttachmentPayload: AttachmentPayload {
```

Comment:
adding support for AudioAttachments looks very interesting and a nice addition to the SDK, would you mind opening a separate PR for this?

Reply:
of course, I'll remove it when the draft is ready for review